### PR TITLE
Store server JSON in NovaServer

### PR DESCRIPTION
**Pull request by manishtomar**
This will be used when storing servers in cache at the succesful convergence cycle. See https://github.com/rackerlabs/otter/pull/1458/files#diff-5894116b577c37059087fcce5ce51693R175

Part of #1458.

### CODE REVIEW

**Comment by radix** [comment]
Could you update the NovaServer docstring to add the new :ivar: for json, pointing out it's to maintain the originally received json?

**Comment by radix** [comment]
I think we may also want to update `otter.convergence.service.server_to_json` to return the `json` attribute. That's used to populate the `active` list for convergence tenants, and currently it only returns a rather useless {'id': server_id} dict.

It should be a pretty simple change, but if you think it should be done in a separate PR that's fine. I can even do it real quick.

**Comment by radix** [comment]
LGTM after the docstring  fix :)

**Comment by manishtomar** [comment]
> I think we may also want to update otter.convergence.service.server_to_json to return the json attribute. That's used to populate the active list for convergence tenants, and currently it only returns a rather useless {'id': server_id, 'links': ...} dict.

@radix I think that should remain the same to be consistent with worker. Also, note that that dict is returned as part of `/state` call which should not change

**Comment by radix** [comment]
Hm, I guess I was confused about what the legacy `active` list contains. I thought it contained the server data. I just tracked it down and I *think* it contains these keys:

- `id`: server id
- `links`: links (we already support this in convergence)
- `name`: we don't have this yet
- `lb_info`: some LB stuff
- `created`: time created

We don't have `name`, `lb_info`, or `created` in new convergence stuff. I guess that's not important (especially since we don't document these keys anywhere :smile:)

anyway forget about my comment :)

File: otter/convergence/model.py [hunk 1]
```diff
@@ -216,9 +217,10 @@ class NovaServer(object):
         the server is on the ServiceNet network
     :ivar str image_id: The ID of the image the server was launched with
     :ivar str flavor_id: The ID of the flavor the server was launched with
-
     :ivar PSet desired_lbs: An immutable mapping of load balancer IDs to lists
         of :class:`CLBDescription` instances.
+    :var dict json: JSON dict received fro mNova from which this server
```
Comment (radix) [inline otter/convergence/model.py]: sorry, you have a typo here :)

**Comment by radix** [comment]
LGTM

File: otter/convergence/model.py (hunk 1 above)
Comment (manishtomar) [inline otter/convergence/model.py]: Oh sorry. Thanks :)

**Pull request merged**